src/scrapers/visa-cal.js

```diff
@@ -145,19 +147,19 @@ function prepareTransactions(txns, startMoment, combineInstallments) {
 
 async function getBankDebits(authHeader, accountId) {
   const bankDebitsUrl = getBankDebitsUrl(accountId);
-  return fetchGet(bankDebitsUrl, authHeader);
+  return fetchGet(bankDebitsUrl, Object.assign(authHeader, HEADER_SITE));
```

Comment:
Make sure you update the original authHeader as following:
1. revert this change so you will pass `authHeader` as is.
2. change line in method `fetchData` line 274 from 
```
    const authHeader = { Authorization: this.authHeader };	
```
 to 
```
    const authHeader = Object.assign({ Authorization: this.authHeader }, HEADER_SITE);
```



**NOTE**: This comment is relevant to all places where you changed `authHeader` into `Object.assign(authHeader, HEADER_SITE)`.